src/components/forms/common/PersonSelect.tsx

```diff
@@ -0,0 +1,96 @@
+import { Autocomplete } from 'mui-rff';
+import { useIntl } from 'react-intl';
+import { useQuery } from 'react-query';
+import { Avatar, Box, Typography } from '@material-ui/core';
+import { FunctionComponent, useEffect, useState } from 'react';
+
+import getPeopleSearchResults from 'fetching/getPeopleSearchResults';
+import useDebounce from 'hooks/useDebounce';
+import { ZetkinPerson } from 'types/zetkin';
+
+
+interface PersonSelectProps {
+    label?: string;
+    name: string;
+    onChange: (person: Partial<ZetkinPerson>) => void;
+    orgId: string | number;
```

Comment:
I think this makes more sense to just get internally using `router.query`, since if this moves in to a component without the `orgId`, then you have to add that prop to the parent OR do this anyway.

Reply:
I guess you're right! I'm not yet comfortable with this approach to objects retrieving their dependencies rather than having them injected, and I do think it complicates testing (in this case `useRouter()` will need to be mocked to test `PersonSelect`) but I'm willing to keep trying and hopefully will feel comfortable soon enough. 😊

Reply:
Actually we don't need to mock `useRouter` im pretty sure, what you do to set values in the router is to use this pattern:

```js
    singletonRouter.query = {
        orgId: '1',
    };
```

We did this in `TaskStatusSubtitle.spec.tsx` if you wanna see it in use

Reply:
Yeah I realized that testing isn't that big of a deal. But I did find another scenario where this was causing extra work, my view thumbnail prototype. But there are always going to be pros and cons that outweigh each other in different situations, so I don't mind!